pages/linux/strigger.md

```diff
@@ -0,0 +1,24 @@
+# strigger
+
+> View or modify Slurm trigger information.
+> More information: <https://slurm.schedmd.com/strigger.html>.
+
+- Register a new trigger. Execute the specified program when the specified event occurs:
+
+`strigger --set --{{primary_database_failure|primary_slurmdbd_failure|primary_slurmctld_acct_buffer_full|primary_slurmctld_failure|...}} --program={{path/to/executable}}`
+
+- Execute the specified program when the specified job terminated:
+
+`strigger --set --jobid={{job_id}} --fini --program={{path/to/executable}}`
```

Comment:
Can you pass arguments too? If so, might be worth an extra example here.

Reply:
The docs state

1. "A hostlist expression for the nodelist or job ID is passed as an argument to the program."
2. "You may quote the path and include extra program arguments if desired."

So I think that you could do

```suggestion
`strigger --set --jobid={{job_id}} --fini --program="{{path/to/executable}} {{argument1 argument2 ...}}"`
```

Do you think that that's correct?

Reply:
That seems good. Let's do that, @gutjuri.